LDAR_Sim/testing/unit_testing/test_programs/test_method/method_testing_fixtures.py

```diff
@@ -641,3 +664,299 @@ def create_random_state():
+        mocker.patch.object(site_mock, "get_survey_cost", mock_get_survey_cost)
+        # Add the mock site to the list
+        mock_sites.append(site_mock)
+    mocker.patch.object(DefaultSensor, "detect_emissions", mock_detect_emissions)
+    mocker.patch.object(
+        Method,
+        "_get_average_method_surveys_required",
+        mock_get_average_method_surveys_required,
+    )
+    mocker.patch.object(
+        Method,
+        "_determine_if_site_survey_can_be_completed",
+        mock_determine_if_site_survey_can_be_completed,
+    )
+    mocker.patch.object(
+        Method,
+        "_get_avg_t_bt_sites",
+        mock_average_t_btw_site,
+    )
+    mocker.patch.object(
+        Method,
+        "check_weather",
+        mock_check_weather_t,
+    )
+    mocker.patch.object(
+        Method,
+        "_get_average_survey_time_for_method",
+        mock_get_method_survey_time,
+    )
+    mocker.patch.object(Method, "_get_travel_time", mock_get_travel_time)
+    mocker.patch.object(DefaultSiteLevelSensor, "detect_emissions", mock_detect_emissions)
+    mocker.patch.object(site_mock, "get_detectable_emissions", mock_get_detectable_emissions)
+    mocker.patch.object(Method, "survey_site", mock_survey_site)
+    properties: dict = {
+        pdc.Method_Params.N_CREWS: 5,
+        pdc.Method_Params.SENSOR: {
+            pdc.Method_Params.TYPE: "default",
+            pdc.Method_Params.MDL: 1,
+            pdc.Method_Params.QE: {
+                pdc.Method_Params.QUANTIFICATION_PARAMETERS: [0.0, 0.0],
+                pdc.Method_Params.Q_TYPE: "default",
+            },
+        },
+        pdc.Method_Params.DEPLOYMENT_TYPE: "mobile",
+        pdc.Method_Params.MAX_WORKDAY: 8,
+        pdc.Method_Params.CONSIDER_DAYLIGHT: False,
+        pdc.Method_Params.T_BW_SITES: {pdc.Common_Params.VAL: [1]},
+        pdc.Method_Params.IS_FOLLOW_UP: False,
+        pdc.Method_Params.WEATHER_ENVS: {
+            pdc.Method_Params.WIND: [0, 10],
+            pdc.Method_Params.TEMP: [-30, 30],
+            pdc.Method_Params.PRECIP: [0, 1],
+        },
+        pdc.Method_Params.REPORTING_DELAY: 0,
+        pdc.Method_Params.COST: {
+            pdc.Method_Params.UPFRONT: 1000,
+            pdc.Method_Params.PER_SITE: 0,
+            pdc.Method_Params.PER_DAY: 5,
+        },
+    }
+    weather = create_random_state()
+    mocker.patch.object(DaylightCalculatorAve, "__init__", mock_daylight_calculator_ave_init)
+    daylight: DaylightCalculatorAve = DaylightCalculatorAve()
+
+    survey_plans = [
+        ScheduledSurveyPlanner(
+            site,
+            6,
+            date(2023, 1, 1),
+            date(2025, 12, 31),
+            [2023, 2024, 2025],
+            [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12],
+        )
+        for site in mock_sites
+    ]
+    workplan = Workplan(survey_plans, date=date(2023, 1, 1))
+    expected_deployment_stats = CrewDeploymentStats(5.0, 1, 2, 120)
```

Comment:
Reading the test data I think this should be 10.0, 5, 10, 600 should it not? As you have 5 sites?

Reply:
With each site being 2 hours to survey, 1 crew shouldn't be able to survey all 5 sites in 8 hours